tests/integration/ha_tests/helpers.py

```diff
@@ -488,3 +517,116 @@ async def verify_writes(ops_test: OpsTest) -> int:
             total_expected_writes == actual_writes
         ), f"{role} {unit.name} missed writes to the db."
     return total_expected_writes
+
+
+async def get_other_mongodb_direct_client(ops_test: OpsTest, app_name: str) -> MongoClient:
+    """Returns a direct mongodb client to the second mongodb cluster."""
+    unit = ops_test.model.applications[app_name].units[0]
```

Comment:
I take it this function is somehow used in the network tests. If the network is down on `ops_test.model.applications[app_name].units[0]` then you won't be able to run `unit.run_action("get-password")` -- in the VM charm I believe we filter by available units

Reply:
or maybe you can pass `active_unit`

Reply:
It's used only for the `test_unique_cluster_dbs()` to get a client to the second cluster, so that entries can be compared.